BUILD.bazel

```diff
@@ -81,6 +82,11 @@ py_binary(
     srcs = [".grabl/wait-for-release.py"],
 )
 
+checksum(
+    name = "checksum-mac",
+    archive = ":mac-zip",
```

Comment:
The file that we should calculate checksum for is `workbase.dmg` right? Does specifying `archive` do it?

Reply:
@lriuui0x0 yes; `checksum` rule unpacks the archive: https://github.com/graknlabs/bazel-distribution/blob/master/common/checksum.bzl#L24